tests/functional/test_freeze.py

```diff
@@ -258,41 +259,31 @@ def test_freeze_mercurial_clone(script, tmpdir):
     _check_output(result, expected)
 
 
-@pytest.mark.network
 def test_freeze_bazaar_clone(script, tmpdir):
     """
     Test freezing a Bazaar clone.
 
     """
-
-    checkout_path = local_checkout(
-        'bzr+http://bazaar.launchpad.net/%7Edjango-wikiapp/django-wikiapp/'
-        'release-0.1',
-        tmpdir.join("cache"),
-    )
-    # bzr internally stores windows drives as uppercase; we'll match that.
+    try:
+        checkout_path = _create_test_package(script, vcs='bazaar')
+    except OSError as e:
+        pytest.skip('Invoking `bzr` failed: %s' % e)
```

Comment:
@pfmoore: Do you agree with this? If so, I'll change it as @xavfernandez suggests.